cmake_ocl.sh

```diff
@@ -31,9 +31,14 @@ build()
 	mkdir ${DIR_NAME}
 	cd ${DIR_NAME}
 
-	cmake -DCMAKE_BUILD_TYPE=Release ${BUILD_OPTIONS} ${CMAKE_OPTIONS} ..
+	CXX=`which CC` CC=`which cc` cmake -DCMAKE_BUILD_TYPE=Release ${BUILD_OPTIONS} ${CMAKE_OPTIONS} ..
```

Comment:
This needs some generic solution... 
We could remove the variables before `cmake` and call the build script like this
```
CXX=`which CC` CC=`which cc` cmake_ocl.sh # etc..
```